docs/index.rst

```diff
@@ -87,6 +87,15 @@ Solidity Tools
 * `evmdis <https://github.com/Arachnid/evmdis>`_
     EVM Disassembler that performs static analysis on the bytecode to provide a higher level of abstraction than raw EVM operations.
 
+Solidity Parsers and Grammars
```

Comment:
Perhaps change to `Third-Party Solidity Parsers and Grammars` or `Parser Tools` otherwise it might be understood as **the** Solidity parser.